app/styles/icomoon.css

```diff
@@ -28,7 +28,7 @@
   text-transform: none;
   line-height: 1;
   -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  -moz-osx-font-smoothing: var(--color-gray-1);
```

Comment:
Does this work?

Reply:
No, this does not work, as the variable sheet is not imported. Revertd back to grayscale asdoes not impact darkMode in any way.